crates/bevy_color/src/color_range.rs

```diff
@@ -6,7 +6,7 @@ use crate::Mix;
 /// end point which must be in the same color space. It works for any color type that
 /// implements [`Mix`].
 ///
-/// This is useful for defining gradients or animated color transitions.
+/// This is useful for defining simple gradients or animated color transitions.
```

Comment:
What is the motivation for adding the word "_simple_" here? I don't see any reason why this trait couldn't be the interface for _any_ gradient type, since it only requires access to `Mix` and the ability to sample the value.